pkg/features/kube_features.go

```diff
@@ -603,15 +603,6 @@ const (
 	// sandbox creation and network configuration completes successfully
 	PodReadyToStartContainersCondition featuregate.Feature = "PodReadyToStartContainersCondition"
 
-	// owner: @wzshiming
-	// kep: http://kep.k8s.io/2681
-	// alpha: v1.28
-	// beta: v1.29
-	// GA: v1.30
```

Comment:
@wzshiming next time please remove code using the feature gate when promoting to GA.

Reply:
Okay, get it, thanks.